tests/unit_and_functional/test_cli.py

```diff
@@ -219,7 +218,7 @@ def test_deployment_test_locally_option_calls_run_workflow_handler(
     )
     deployment(args)
     expected_pass_through_args = Namespace(
-        namespace="foo1",
+        namespace="bodywork-deployment-jobs",
```

Comment:
We shouldn't need this anymore, because we shouldn't be passing it through.

Reply:
Yup, that evaded the cleanup as did the one below